prism/src/prism/PrismSettings.java

```diff
@@ -248,6 +252,8 @@
 																			"Epsilon value to use for checking termination of iterative numerical methods." },
 			{ INTEGER_TYPE,		PRISM_MAX_ITERS,						"Termination max. iterations",			"2.1",			new Integer(10000),															"0,",																						
 																			"Maximum number of iterations to perform if iterative methods do not converge." },
+			{ INTEGER_TYPE,		PRISM_PRECISION,						"Floating point precision",			"2.1",			new Integer(64),															"1,",																						
+																			"The accuracy of the mantissa (in bits) is determined by this variable. This user-selectable precision is a minimum value: it is rounded up to a whole limb." },
```

Comment:
For the version number, if I understood the procedure correctly, you should pick the last publicly released version number before the setting was introduced. This applies to the other newly added settings as well.